task_lib.c

```diff
@@ -28,7 +28,9 @@
 
 #include "ph.h"
 #include "task_lib.h"
-#include "stewticket_thr.h"
+#include "ticketlock.h"
```

Comment:
Please merge with the latest updates.  We're using `stewticket_thr.h` now with `USE_TICKET_LOCKS` defined to provide us with a thread friendly implementation of ticket locks, which in practise is almost as fast as the pthread MP locks we were using before, but behave way more consistently when a thread can't get a hold of a CPU quickly.  The threaded ticketlock mechanism prevents CPU starvation, and can reduce the incidence of request/response timeouts, and improves timer mechanism consistency.